libs/domain/picking/src/components/index.ts

```diff
@@ -1,4 +1,6 @@
 export * from './customer-note/customer-note.def';
+export * from './filter-button/filter-button.def';
+export * from './filters/filters.def';
```

Comment:
we need to export the component as well:
- add an index.ts to each component dir
- export the components, styles, def, etc in each barrel
- export the barrels in this component barrel

Reply:
My bad, we should not export def file in the main barrel as it will break lazy loading of the components.